package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@shopware-ag/admin-extension-sdk",
   "license": "MIT",
-  "version": "0.0.52",
+  "version": "0.0.53",
```

Comment:
I think this is breaking. Therefore I would say we should increase it to `1.0.0`. And I think it could be a good idea to add a changelog file until now